design/view-change.md

```diff
@@ -180,6 +180,28 @@ we don't place any upper limit on this time.
   complex request selection algorithm can be simplified and view change
   algorithm will still work.
 
+- All above is applied to just one instance of PBFT protocol, but RBFT is
+  basically multiple PBFT instances running in parallel, with only master
+  altering state. RBFT states that all instances on given node should start
+  view change simultaneously and each instance should elect new primary
+  independently. Since indy plenum implementation has simple round robin
+  election it should be ok to just run view change on master with each backup
+  just waiting for view change completion on master.
```

Comment:
So, do we have some special messages indicating that View Change is finished (like ViewChangeDone in the current protocol), and replicas on backup instance do not order anything new until they receive a quorum of ViewChanageDone from master?
Or do we just have some local state across all instances indicating that view change is finished on master and backup instances can start ordering?

Reply:
@ashcherbakov PBFT has NEW-VIEW message which is sent by just primary but it is aknowledged by quorum of previously received VIEW-CHANGE messages, and if primary sends different NEW-VIEW messages to different replicas it's guaranteed that the'll spot the difference.